src/cookbook/design/themes.md

```diff
@@ -9,42 +9,78 @@ js:
 
 <?code-excerpt path-base="cookbook/design/themes"?>
 
+{{site.alert.note}}
+  This recipe uses Flutter's support for [Material 3][] and
+  the [google_fonts][] package.
+{{site.alert.end}}
+
+[Material 3]: https://m3.material.io
+[google_fonts]: {{site.pub-pkg}}/google_fonts
+
 To share colors and font styles throughout an app, use themes.
-You can either define app-wide themes, or use `Theme` widgets
-that define the colors and font styles for a particular part
-of the application. In fact,
-app-wide themes are just `Theme` widgets created at
-the root of an app by the `MaterialApp`.
 
-After defining a Theme, use it within your own widgets. Flutter's
-Material widgets also use your Theme to set the background
-colors and font styles for AppBars, Buttons, Checkboxes, and more.
+You can define app-wide themes, add [`Theme`][] widgets
+to other component widgets, or both.
+Each theme defines the colors, type style, and other parameters applicable
+for the type of Material component.
+
+Many Material 3 widgets have their own `Theme` widget and
+related `ThemeData` data class.
+The `Theme` widget itself applies to the whole `MaterialApp`.
+
+For example: [`TextButton`][] can use both [`TextButtonTheme`][] to apply a theme
+and [`TextButtonThemeData`][] to define the visual properties of that theme.
+Unlike the general app theme, the `TextButtonThemeData` can include the button's
+width, height, padding, and other positioning choices.
+
+[`TextButton`]: {{site.api}}/flutter/material/TextButton-class.html
+[`TextButtonTheme`]: {{site.api}}/flutter/material/TextButtonTheme-class.html
+[`TextButtonThemeData`]: {{site.api}}/flutter/material/TextButtonThemeData-class.html
+
+If a widget doesn't use a specific theme, the visual properties fall back
+to the main theme. Precedence in themes rank from the specific widget,
+to the widget type theme, to the general theme.
```

Comment:
Not super clear

```suggestion
Typically, visual properties are resolved first with the.
widget properties (e.g. `AppBar.centerTitle`). If not specified, then the widget theme (e.g. `AppBarTheme.centerTitle` of the ambient theme) is used. If not specified, then some
reasonable default is used.
```

Reply:
This copy doesn't read clearly. Stet.

Reply:
@atsansone I also think it's important to handle this clarification before this lands. It's not clear what "If a widget doesn't use a specific theme" means and the existing text doesn't account for the properties on the widgets themselves as `@guidezpl` highlighted in his suggestion.